src/node/rpc/memberfrontend.h

```diff
@@ -120,6 +120,21 @@ namespace ccf
            u->put(id, {cert});
            return true;
          }},
+        {"set_user_data",
+         [this](Store::Tx& tx, const nlohmann::json& args) {
+           const UserId user_id = args["user_id"];
```

Comment:
Should the JSON `args` be cast to a suitable `struct` instead of addressing the JSON fields directly?